install_config/web_console_customization.adoc

```diff
@@ -68,6 +68,36 @@ link:https://github.com/openshift/origin-web-console/tree/master/extensions/exam
 Origin] repository on GitHub.
 ====
 
+[[setting-extension-properties]]
+=== Setting Extension Properties
+
+You can pass settings from the *_master-config.yaml_* file to be used by the
+extension using the
+xref:../install_config/master_node_configuration.adoc#master-config-asset-config[`extensionProperties`
+mechanism], which appears in a generated configuration when using
+`--write-config`:
+
+----
+assetConfig:
+  extensionDevelopment: true
+  extensionProperties:
+    doc_url: https://docs.openshift.com
+    hello: world
+    test: thing
+  extensionScripts:
+----
+
+This results in a global variable that can be accessed by the extension:
+
+----
+window.OPENSHIFT_EXTENSION_PROPERTIES = {
```

Comment:
Will this be syntax-highlighted as JavaScript?

Also, we may want to say "results in a global variable that can be accessed by the extension, as if the following code was executed:"

(it's a JSON object stored in the global variable `OPENSHIFT_EXTENSION_PROPERTIES`; in JavaScript, `window` is the global context)